x-pack/plugin/inference/qa/inference-service-tests/src/javaRestTest/java/org/elasticsearch/xpack/inference/DefaultElserIT.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference;
+
+import org.elasticsearch.inference.TaskType;
+import org.elasticsearch.threadpool.TestThreadPool;
+import org.elasticsearch.xpack.inference.services.elser.ElserInternalService;
+import org.hamcrest.Matchers;
+import org.junit.After;
+import org.junit.Before;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.oneOf;
+
+public class DefaultElserIT extends InferenceBaseRestTest {
+
+    private TestThreadPool threadPool;
+
+    @Before
+    public void createThreadPool() {
+        threadPool = new TestThreadPool(DefaultElserIT.class.getSimpleName());
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        threadPool.close();
+        super.tearDown();
+    }
+
+    @SuppressWarnings("unchecked")
+    public void testInferCreatesDefaultElser() throws IOException {
+        assumeTrue("Default config requires a feature flag", DefaultElserFeatureFlag.isEnabled());
+        var model = getModel(ElserInternalService.DEFAULT_ELSER_ID);
+        assertDefaultElserConfig(model);
+
+        var inputs = List.of("Hello World", "Goodnight moon");
+        var queryParams = Map.of("timeout", "120s");
+        var results = infer(ElserInternalService.DEFAULT_ELSER_ID, TaskType.SPARSE_EMBEDDING, inputs, queryParams);
+        var embeddings = (List<Map<String, Object>>) results.get("sparse_embedding");
+        assertThat(results.toString(), embeddings, hasSize(2));
+    }
+
+    public void testScaleFrom0() {
+        assumeTrue("Default config requires a feature flag", DefaultElserFeatureFlag.isEnabled());
+        // fail("how can we test this when we need to wait for the cooldown");
```

Comment:
Scaling up from 0 should be quite fast (few seconds).

You could do:
- start with zero allocations
- perform request -> assert 409 conflict
- poll a get inference or trained model endpoint until num_allocations=1 (or 10secs elapse)
- perform request -> assert 200 success

Scaling down to 0 is slow (15 minutes) and it better suited for a QA test.

Reply:
++ I'll create this test in a follow up PR that adds support for scale from 0